configs/config.ini

```diff
@@ -158,6 +158,11 @@ path_scenario = scenario/migrate.yaml
 # rollback.
 mysqldump_host = <specific MySQL DB hostname or IP address, not VIP>
 
+# Uses low-level DB requests if set to True, may be incompatible with more
+# recent versions of Keystone. Tested on grizzly, icehouse and juno. True by
+# default
+optimize_user_role_fetch = [True|False]
+
```

Comment:
We still have one more config template in `devlab`... Maybe we need to add this option there too?